backend/src/models/attendee.js

```diff
@@ -0,0 +1,12 @@
+const mongoose = require('mongoose')
+
+const Attendee = new mongoose.Schema({
+  name: { type: String, required: true },
+  netid: { type: String, required: true },
+  year: { type: Number },
+  isCandidate: { type: Boolean },
```

Comment:
Instead of `isCandidate` I would have this field be the `_id` field from the candidate. If it is null, we can assume that they are not a current candidate.